### PR TITLE
version upgrades

### PR DESCRIPTION
**Pull request by nicholas-goodwin**

### CODE REVIEW

**Comment by nicholas-goodwin** [comment]
Couldn't get much in way of getting the tests to run in GitHub actions, so here: 
Screenshot 2023-06-02 at 3 37 17 PM

File: lib/elastix/mapping.ex [hunk 1]
```diff
@@ -45,29 +45,18 @@ defmodule Elastix.Mapping do
...
           index_names :: String.t() | list,
-          type_names :: String.t() | list,
           query_params :: Keyword.t()
         ) :: HTTP.resp()
-  def get(elastic_url, index_names, type_names, query_params \\ [])
+  def get(elastic_url, index_names, query_params \\ [])
 
-  def get(elastic_url, index_names, type_names, query_params)
-      when is_list(type_names) and is_list(index_names) do
-    prepare_url(elastic_url, make_path(index_names, type_names, query_params))
```
Comment (c-m-duncan) [inline lib/elastix/mapping.ex:55]: I guess all these aren't needed anymore?

**Review by c-m-duncan — APPROVED** [review]
lgtm

File: lib/elastix/mapping.ex (hunk 1 above)
Comment (nicholas-goodwin) [reply lib/elastix/mapping.ex:55]: as part of elasticsearch bumping past 7.0 (I think?), type names are not needed for sending in the request, as, from what I've seem, the endpoint is completely based on the index

**Pull request merged**